docs/learn/platformio.md

```diff
@@ -0,0 +1,77 @@
+# PlatformIO
+
+[PlatformIO](https://platformio.org/) is another way to prepare firmware (amongst other things) to use on your control board after making any changes or updates. Previously most of us had used ArduinoIDE but PlatformIO is helping to get around some of Arduino's 32bit compile limitiations. Luckily in the end only one single line in the firmware needs to get changed to make it work, and I will be making that change, and you will now be free to decide which you want to use. Even better is this is **cross platform**!
+
+---
+
+## Installation
+
+I will write this from a Windows perspective, as far as I know it will look very similar for MAC and Linux. If it does not please help out and add some pictures to clearify.
+
+### Install 1 - VS Code
+
+The first step is to install Microsoft Visual Studio Code. This is **cross platform** and it is a very simple, yet powerful tool. I have used this for a while as my text editor for programming / coding. Previously I tried Atom and Notepad++, this has been more intuitive for me.
+Install [VS Code](https://code.visualstudio.com/) on your machine. Download the appropriate version from this link [VS Code](https://code.visualstudio.com/), and follow the prompts. This part should be fairly simple and staright forward. 
+
+### Install 2 - PlatformIO
+
+PLatformIO is actually and "Extension" for VS Code. To install PlatformIO you click on the broken block icon on the left side of VSCODE shown here highlighted.
+
+![!VS Code Extension](https://www.v1engineering.com/wp-content/uploads/2019/12/Extension.jpg){: width="400"}
+
+This will pop up a search window. Type in "PlatformIO". It should be the first result, now just click the green install button.
+
+Now the difficult part....you have to wait for it to finish. It can take up to 15 minutes on an older machine.
+
+![!VS Code Waiting](https://www.v1engineering.com/wp-content/uploads/2019/12/Still_installing.jpg){: width="400"}
+
+At the very bottom of the VS Code window you will see a purple bar and the installing icon and message, touch nothing while it works. When it is done it will pop up a restart button.
```

Comment:
```suggestion
At the very bottom of the VS Code window you will see a purple bar and the installing icon and message, *don't touch anything* while it works. When it is done it will pop up a restart button.
```